src/prefect/config.toml

```diff
@@ -38,6 +38,8 @@ format = "[%(asctime)s] %(levelname)s - %(name)s | %(message)s"
 # Send logs to Prefect Cloud
 log_to_cloud = false
```

Comment:
I think maybe both `log_to_cloud` and `heartbeat` should be in a subsection of `logging` called `cloud`, as they only apply in a cloud context.

Reply:
Discussed offline, I'll move `heartbeat` to `logging_heartbeat` inside the `[cloud]` section